src/test/scala/it/pagopa/interop/catalogprocess/CatalogProcessSpec.scala

```diff
@@ -216,23 +216,38 @@ class CatalogProcessSpec extends SpecHelper with AnyWordSpecLike with BeforeAndA
           certified = List(
             CatalogManagementDependency
               .Attribute(
-                single =
-                  Some(CatalogManagementDependency.AttributeValue("0001", explicitAttributeVerification = false)),
+                single = Some(
+                  CatalogManagementDependency.AttributeValue(
+                    UUID.fromString("a54aebcc-f469-4c5a-b232-8b7003824300"),
```

Comment:
how about using the variables here (and in the following lines) to make it more clear?